src/App.jsx

```diff
@@ -104,7 +106,8 @@ class App extends Component {
 
   render() {
     const { classes, theme } = this.props;
-    const { projectSelected, allProjects } = this.state;
+    const { allProjects } = this.state;
```

Comment:
What do you think about renaming `allProjects` to `projects`.

Reply:
okay! it is inexistent in the next PR but should I need it again I will change it to `projects`